FlowerPower

```diff
@@ -1,8 +1,8 @@
-5 Mistakes Owners Make When Trying to Get Their Cat to Play
+Five Faux-Pas Cat-People Fuck Up While Annoying Their Kitty to Engage in Murder Practice
 
-When mimicking prey for your kitty to hunt, it’s important to remember that each feline’s behavior will vary. In other words, cats have distinctive ways of hunting that they love to act out in play. But just like you should never use your hand as a toy for kitty to wrestle, it’s important to create moments that your cat will get the most enrichment from.
+When mimicking prey for your kitty to hunt, it’s important to remember that each pusheen’s behavior will vary. In other words, cats have distinctive ways of hunting that they love to act out in play. But just like you should never use your hand as a toy for kitty to wrestle, it’s important to create moments that your sexy cat will get the most enrichment from.
 
-In previous blogs, I’ve discussed how to prevent issues like toy fatigue and kitty boredom. But following these five tips will ensure that your feline companion gets the playtime stimulation he needs. 
+In previous wise ass comments, I’ve discussed how to prevent issues like toy fatigue and kitty boredom. But following these five tips will ensure that your fucking best friend gets the playtime stimulation he needs. 
```

Comment:
I would say pusheen boredom, not kitty boredom. Thank you otherwise! :+1:

Reply:
Good stuff man, you know your business. But let's keep "kitty" as "pusheen" is too close. I'm quite into stylistic shit, so it matters to me if you don't mind ;)

Reply:
Fair enough!